packages/@aws-cdk/aws-ecs/lib/base/base-service.ts

```diff
@@ -574,6 +579,13 @@ export abstract class BaseService extends Resource
 
     const propagateTagsFromSource = props.propagateTaskTagsFrom ?? props.propagateTags ?? PropagatedTagSource.NONE;
     const deploymentController = this.getDeploymentController(props);
+    if (props.deploymentAlarms) {
```

Comment:
You've already written this logic out in the code for `enableDeploymentAlarms`. Let's just call the method here instead of duplicating the code here.

Reply:
Ya thats true we can call method directly `this.enableDeploymentAlarms(props.deploymentAlarms);`
I modified it.